bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Voice.java

```diff
@@ -339,10 +340,10 @@ public static void listenAndAnswer(@ParamDoc(name = "source") @Nullable String s
      *
      * @param stt the speech-to-text service to use or null to use the default service
      * @param tts the text-to-speech service to use or null to use the default service
-     * @param interpreter the human language text interpreter to use or null to use the default service
+     * @param interpreter  comma separated list of human language text interpreters to use or null to use the default services
```

Comment:
Please rename the parameter into `interpreters`
... or null to use the default **service**